### DISCUSSION TITLE
Onde você procura vagas de frontend/webdev?

### DISCUSSION THREAD

**Discussion by PabloDinella** [frontendbr/forum#15] [Discussão]
Estou na busca de vagas (de estágio) em frontend, os sites (que me vêm à mente agora) em que eu busco são:

http://trampos.co/
http://ondetrabalhar.com/
http://ceviu.com.br/
http://superestagios.com.br/

outros que não mostram as vagas diretamente (ou com pouca atualização):

http://contratado.me/
http://workajobs.com.br/

Poste aí se conhece algum outro :)

Mais uma coisa: que tal um repositório pra postar vagas via issues?

**Comment by tcelestino** [comment]
http://linkedin.com
http://empregos.com.br
http://vagas.com.br

**Comment by felipefialho** [comment]
@pablodinella, curti sua ideia.

O que acham pessoal?

@diogomoretti @vitorbritto @fdaciuk @eduardojmatos @Mendrone @willianjusten 

Aliás, como chamo toda a equipe sem precisar escrever um por um mesmo? rs

**Comment by bernardodiasc** [comment]
Mais um link https://github.com/lerrua/remote-jobs-brazil

@LFeh https://github.com/blog/1121-introducing-team-mentions :)

**Comment by bernardodiasc** [comment]
Fiz parte de um grupo de trabalho que usava o proprio repo pra guardar as paradas (varios docs, arquivos de código, etc), tanto quanto issues pra discussões e wiki pra documentações oficiais da ferramenta. Ta abandonado mas fica de inspiração https://github.com/symphonycms/wg, da pra centralizar muita coisa num repo só, tem até um livro nesse wg que comentei...

**Comment by lucasbento** [comment]
@bernardodiasc +1 pelo link do repo de remote-jobs!

**Comment by eduardojmatos** [comment]
@LFeh apoio 100%!!

**Comment by eduardojmatos** [comment]
@lucasbsilva boa ideia!! Vou criar um time da galera da org pra mencionar aqui!

---

update: vlw @bernardodiasc pela dica dos teams :heart:

**Comment by eduardojmatos** [comment]
@lucasbsilva olhae!

@frontendbr/admins testando, 1, 2, 3 :heart:

**Comment by felipefialho** [comment]
Funcionou \o

E esse repo de vagas na nossa org, o que acham?

Podemos receber vagas por pull-request ou até por email caso o responsável não use Github (rh e tal).

O bom é que podemos fazer uma curadoria.

**Comment by lucasbento** [comment]
@LFeh um repo de vagas seria animal!

**Comment by eduardojmatos** [comment]
@LFeh eu apoio isso mano. De repente a gente faz um template pra galera que quer postar.

Se não seguir o template certinho, a gente não insere. O Felquis fazia isso num grupo de vagas, no Facebook, se não me engano.

De repente, pro pessoal que não manja de código/git, podemos criar um esquema à parte pra add essas vagas, como você sugeriu @LFeh (email, etc.).

---

O @dancasttro falou de um projeto dele, mas vi aqui que tá uns 3 meses parado e o site não tá rolando no gh-pages. De repente você pode nos ajudar hein @dancasttro? O que acha? Criamos o repo, você e a galera nos ajuda na estrutura, docs e etc!
- link do projeto => https://github.com/front-jobs/front-jobs.github.io

**Comment by lucasbento** [comment]
Me interessei bastante pelo projeto do @dancasttro, seria bem legal iniciar algo do tipo aqui

**Comment by dancasttro** [comment]
Estou dentro

**Comment by dancasttro** [comment]
podem criar o repo que a gente integra o projeto na org. O projeto esta bem simples mas da pra começar a fazer alguma coisa

**Comment by vitor-mendrone-deel** [comment]
@eduardojmatos @LFeh Concordo 100% com a criação do repo, vai ser animal, principalmente a curadoria!

**Comment by PabloDinella** [comment]
Que tal um projeto como o do @dancasttro, com gh-pages, onde as vagas são enviadas como PR pra uma pasta /jobs/, e são exibidas num loop do jekyll?
Talvez até o próprio projeto do @dancasttro funcionava assim

**Comment by diogomoretti** [comment]
Seria bacana um repo /vagas ou /jobs, ai deixariamos somente as vagas do Brasil.

**Comment by dancasttro** [comment]
No projeto que fiz eu uso o firebase

**Comment by vitorbritto** [comment]
Sites de vagas é algo frustrante. Sempre tem algum cadastro para ser feito antes de ver **aquele contato mágico** da empresa ou um plano para pagar e manter o candidato "em destaque". 

É um sistema justo, porém complicado. Essa disputa faz com que o possível candidato perca muito tempo do seu dia tentando algo interessante. Na maioria das vezes, é uma facada na expectativa e frustração explodindo para todos os lados. Eu não tive quaisquer experiências interessantes nestes sites. 

Sou adepto de pesquisar as redes sociais da empresa/agência, analisar os jobs que já fizeram e entender a metodologia que utilizam. Se valer a pena, envio um currículo, perfil do GitHub ou LinkedIn e faço perguntas via e-mail ou outro meio de comunicação. É um trabalho difícil, mas vale muito a pena! Afinal, se não correr atrás... fica para trás. =P

**Sobre o repositório de Vagas**

Sou a favor de abrir um repositório para vagas, mas precisamos definir os requisitos e restrições para as vagas serem publicadas. BTW, a API do GitHub está aí. =]

**Comment by PabloDinella** [comment]
@vitorbritto é por isso que separei os links que não mostram as vagas diretamente, acho isso de se cadastrar uma chatice também.

Algo pior ainda é quando as empresas não respondem o e-mail. Perde-se tempo redigindo algo pra não ficar enviando só cópia, e a empresa nem pra responder dizendo um "obrigado pelo interesse".

**Comment by lazarofl** [comment]
A ideia de fazer o pull request numa pasta é uma boa, pensei num formato desses recentemente pra vagas em React. Um formato mais "simples" é criar o projeto e cada vaga é publicada como uma issue, assim é possível fechar a issue quando a vaga não estiver mais disponível, teria um índice de vagas ativas e etc.
A vantagem desse formato é que mantem o controle do status da vaga, a desvantagem é que fica um campo aberto, sabe-se la o que a galera vai preencher. O que acham?

**Comment by PabloDinella** [comment]
Acho que temos que migrar essa discussão do repo de vagas pra outro lugar :) Bora criar um repo já pras vagas e lá continuamos discutindo se vai ser por issues ou PR?

**Comment by lazarofl** [comment]
Por mim ok, acho que dependendo do caso podemos acabar criando um bom projeto a partir desse problema que as empresas possuem, um generator de vagas bem interessante, a empresa poderia dar um fork e subir a sua própria pagina de vagas disponíveis pela conta dela no github, ou executar o generator para montar o repositório para ela usar como gh-page. Deveras útil esse paranauê.

**Comment by lazarofl** [comment]
Se for algo mais controlado via um único local q disponibiliza as vagas a ideia do PR ou issue seria o caso.

**Comment by felipefialho** [comment]
@frontendbr/admins A ideia é abrir o repo dentro da nossa org? Assim como "forum" teria um "vagas"?

**Comment by diogomoretti** [comment]
Concordo @LFeh, ai podemos abrir as vagas nas issues mesmo. Ai quando a vaga for preenchida, o cara fecha a issue. Só teríamos que ter um "template" inicial para as vagas!

**Comment by felipefialho** [comment]
Perfeito, essa é a ideia.

**Comment by mustbetommy** [comment]
+1 Gostei da ideia do repo para as vagas.

**Comment by vitor-mendrone-deel** [comment]
@LFeh @diogomoretti @vitorbritto excelente ideia. Vamos criar o repo e o template para as vagas. O importante é definir quais informações serão obrigatórias pra divulgação delas.

**Comment by mustbetommy** [comment]
Acho que o essencial:
- Área
- Nível de experiência necessária
- Local (se é remoto ou presencial)
- Salário (o valor, ou se é a combinar)
- Uma breve descrição da vaga
- Contatos

**Comment by vitor-mendrone-deel** [comment]
Criei o repositório "vagas", vamos organizando as ideias. PR são bem vindos :smile:

**Comment by lazarofl** [comment]
Acho que cada especialidade poderia ser uma **label**, dessa forma seria possível organizar bem como no exemplo que montei abaixo usando o essencial que o @kevinthomas-dev mencionou e ainda por cima usar a API do github para consultar as issues abertas (vagas abertas) e exibir usando jekyll.
https://github.com/lazarofl/lazarofl.github.io/issues

![image](https://cloud.githubusercontent.com/assets/913314/12784890/e9d168aa-ca70-11e5-90da-2d671af1c807.png)

Como mais uma sugestão para organizar, o local poderia ser atribuído usando **milestones** ao invés de deixar na descrição, assim dá pra trabalhar mais ainda a questão da consulta via github api.

![image](https://cloud.githubusercontent.com/assets/913314/12784935/22cc785c-ca71-11e5-807e-0076f5634751.png)

O que acham?

**Comment by lazarofl** [comment]
Acredito que já deva existir algum projeto que leia as issues para montar uma saída visual usando gh-pages,

**Comment by williamknn** [comment]
Sobre a pergunta inicial do tópico, eu já desisti de procurar trampo em sites de vagas. Além de terem muitos sites que o candidato precisa preencher a vida inteira através de 23468727 formulários, sempre tem candidatos demais para cada única vaga. No trampos, quando você se candidata, aparece embaixo: _"Você e mais a torcida do Curintia se candidataram a esta vaga"_. As empresas nem devem ver todas as solicitações que chega através desses sites.

O que eu uso mesmo é Facebook e LinkedIn. Só.

Sobre o repo, eu gostei da ideia :)

**Comment by bernardodiasc** [comment]
No X-Team, se não me engano, eles fazem entrevista com todo mundo que cadastra no site [https://x-team.com/join/](https://x-team.com/join/?utm_source=bernardodias), mas normalmente é mesmo bem mais comum contratarem por indicação. Entrei por indicação a quase 3 anos atras.

**Comment by willycamargo** [comment]
No caso da minha cidade tem um [site local](http://jobslondrina.com/) para vagas de nossa área. Tem muita vaga ridícula, mas as vezes pinga algumas vagas boas. As empresas também postam diretamente em seus sites e redes sociais, o que facilita muito.

Eu acredito que possa aparecer vagas boas nesses sites, mas elas se perdem ao meio de muitas vagas ruins, o que acaba prejudicando o candidato. Assim como devem haver candidatos bons que se perdem ao meio dos vários candidatos ruins. Ou seja: sistema ruim para os candidatos e para as empresas.

Ao meu ponto de vista as redes sociais tendem a ter melhores oportunidades.

**Comment by felipefialho** [comment]
@lazarofl Curti suas ideias, ficaria bem organizado. Se quiser vai dando uns pull-requests la no repo que criamos. 

Quem sabe não surge alguma coisa que realmente ajude o pessoal a encontrar vagas legais?

**Comment by napalmdev** [comment]
ótima a ideia hein, tem tudo pra certo, e assim a organização vai ficar impecável.

**Comment by lazarofl** [comment]
boa, estou pesquisando alguns formatos antes de sair publicando algo lá, o bacana é postar uma issue nesse repo com alguma proposta e colher o feedback antes, essas discussões de ideias são boas. Dei uma lida nas docs do Jekyll e não possuem suporte a leitura das issues, somente valores mais estáticos mesmo. Ou seja, precisaria de uma linguagem server-side mesmo como node ou ruby lendo da api do github, sem problemas.

**Comment by lazarofl** [comment]
postado uma issue para discutir uma proposta lá dentro da repo.

**Comment by fmoliveira** [comment]
Vagas para trabalho remoto: https://weworkremotely.com/

**Comment by yanmagale** [comment]
Site de vagas tem o http://www.infojobs.com.br/ tb.

Sobre o repositório de vagas, uma palavra: Sensacional

**Comment by DevCarlosAugusto** [comment]
Pra vagas em portugal:
https://www.itjobs.pt/

**Comment by rubenmarcus** [comment]
Existe alguma referência para conseguir trampar remoto?

**Comment by eduardojmatos** [comment]
@rubenmarcus tem um repo do @lerrua bem bacana pra isso:

https://github.com/lerrua/remote-jobs-brazil

É uma lista de empresas que contratam ou dão suporte a trabalho remoto. Dá uma conferida lá que é bacana :smile:
